src/shared/scrapers/DE/DE-BB/index.js

```diff
@@ -3,13 +3,13 @@ import defaultScraperDEU, { sharedSchema } from '../_shared.js';
 
 const scraper = {
   ...sharedSchema,
-  country: 'DEU',
-  state: 'DE-BB', // ISO 3166 notation
+  country: 'iso1:DE',
+  state: 'iso2:DE-BB', // ISO 3166 notation
   scraper: defaultScraperDEU,
   _rowToResult: row => {
     return {
-      cases: parse.number(row[`${scraper.state}_cases`]),
-      deaths: parse.number(row[`${scraper.state}_deaths`]),
+      cases: parse.number(row[`${scraper.state.slice(5)}_cases`]),
+      deaths: parse.number(row[`${scraper.state.slice(5)}_deaths`]),
```

Comment:
Why?

Reply:
Yes, `iso2:` is always 5 characters exactly.

Reply:
I mean those codes will definietly not change. The "perfect" one would be to use country-levels.js / splitId, but this was easier as I could just to a search-replace on the codebase.